### ISSUE TITLE
KSamplerAdvanced Ninja build failed

### ISSUE THREAD

**Issue by huiyu9144** [thrumdev/comfyui-RadialAttention#3]
KSamplerAdvanced
Ninja build failed. Ninja output:
ninja: error: C:\Users\Administrator\.cache\flashinfer\89\cached_ops\batch_prefill_with_kv_cache_dtype_q_f16_dtype_kv_f16_dtype_o_f16_dtype_idx_i32_head_dim_qk_128_head_dim_vo_128_posenc_0_use_swa_False_use_logits_cap_False_f16qk_False\build.ninja:65: expected newline, got ':'
build $name/batch_prefill_paged_kernel_mask_0.cuda.o: cuda_compile C:\Us...
^ near here
ninja: Entering directory `C:\Users\Administrator\.cache\flashinfer\89\cached_ops'

**Comment by rphmeier** [comment]
This is an error in flashinfer (https://github.com/flashinfer-ai/flashinfer). Please refer to the installation procedure for it, it's out of scope for this node. RadialAttention relies on it for optimized masked attention kernels
